compiler/tflite2circle/driver/Driver.cpp

```diff
@@ -55,7 +55,7 @@ int entry(int argc, char **argv)
   }
   catch (const std::runtime_error &err)
   {
-    std::cout << err.what() << std::endl;
+    std::cerr << err.what() << std::endl;
```

Comment:
Please separate `compiler/*` modules in different PRs

Reply:
@seanshpark Yes, I know. It seems implicit rule in this repo. But may I ask why? For who? For what? I personally does not agree the rule. If reviews are pending, and if the changes are conceptually same thing, I believe putting in one PR makes sense.

Reply:
>  But may I ask why? For who? For what? I personally does not agree the rule

This was made by @parjong from the first place when this repo was made(?) to co-work with SRR members not to mixup codes with different projects (SR leading, SRR leading) and as a general rule, make PRs easy to review. This is my understanding what I know. But you may ask to @parjong.

If you don't want to follow this rule, then take the maintainer authority and define your own.

Reply:
> > But may I ask why? For who? For what? I personally does not agree the rule
> 
> This was made by @parjong from the first place when this repo was made(?) to co-work with SRR members not to mixup codes with different projects (SR leading, SRR leading) and as a general rule, make PRs easy to review. This is my understanding what I know. But you may ask to @parjong.
> 
> If you don't want to follow this rule, then take the maintainer authority and define your own.

Everyone needs not to be maintainer. Someone may give his/her opinion. : )

Reply:
> Then, I don't think this case has same problem or issues.

The rule was and still applied without exceptions.
If you want to break this rule, you can go ahead as a maintainer.
As I followed the basic rule from the start, breaking this rule means breaking the decision what build up till now.
For that I cannot go on as a maintainer thus I will leave ONE development.

CC @lemmaa

Reply:
>> Then, I don't think this case has same problem or issues.
>
> The rule was and still applied without exceptions.

I am saying why this case is different that this rule is valid.

- There is no one who working for `one-cmds` except for UX T/F members (@jyoungyun and @jinevening)
- No one from SRR work for this task, and 
- Even you said 5 days ago,

> And as I am not following any activity in current UX related task, so please do not add me as a reviewer,
I don't know the backgrounds thus I can not decide go or no-go.
from https://github.com/Samsung/ONE/pull/7386#issuecomment-894016388

This rule works as a blocker against progress to make things done in given schedule.

> If you want to break this rule, you can go ahead as a maintainer.

I don't want and can't break rule because I am not maintainer.
I am just suggesting my opinion, which works well for onert code, which I had mainly worked for.

Reply:
> I don't want and can't break rule because I am not maintainer.
I am just suggesting my opinion, which works well for onert code, which I had mainly worked for.

These two conflict.
You don't want to break the rules cause you are not maintainer.
You want to break the rule by means of suggestion through me, which I cannot.
So I suggestest you to become a maintainer and build up your own rule with the `compile/*` but you don't want to.


> There is no one who working for one-cmds except for UX T/F members (@jyoungyun and @jinevening)
No one from SRR work for this task, and
Even you said 5 days ago,

These are true. But I don't understand why these can be a reason to mix `compiler/*` modules is OK.
For the worst case, I could say mixing modules makes review hard and still request to split them.

> This rule works as a blocker against progress to make things done in given schedule. (for trivial PRs like this).

You can split this PR into two and that will comply the rule and I don't know how that is a blocker.
The problem I see here is that review itself is SLOW,

Reply:
Personally, I don't like that rule. (I'm not a robot, so I prefer to operate with the appropriate flexibility at a level that can convince each other. ^_^)

However, it is also true that such rules have been maintained here. If several people feel uncomfortable, I think it is necessary to discuss and revise it together. But until then, it's also correct to follow the old rules. So, perhaps it is better to leave this as a `Draft PR`, and after sufficiently discussing and modifying it, split it into several PRs and reflect it.